neqo-transport/src/connection/idle.rs

```diff
@@ -46,20 +46,14 @@ impl IdleTimeout {
         self.timeout = min(self.timeout, peer_timeout);
     }
 
-    pub fn expiry(&self, now: Instant, pto: Duration, keep_alive: bool) -> Instant {
+    pub fn expiry(&self, now: Instant, pto: Duration) -> Instant {
         let start = match self.state {
             IdleTimeoutState::Init => now,
             IdleTimeoutState::PacketReceived(t) | IdleTimeoutState::AckElicitingPacketSent(t) => t,
         };
-        let delay = if keep_alive && !self.keep_alive_outstanding {
-            // For a keep-alive timer, wait for half the timeout interval, but be sure
-            // not to wait too little or we will send many unnecessary probes.
-            max(self.timeout / 2, pto)
-        } else {
-            max(self.timeout, pto * 3)
-        };
+        let delay = max(self.timeout, pto * 3);
         qtrace!(
-            "IdleTimeout::expiry@{now:?} pto={pto:?}, ka={keep_alive} => {t:?}",
+            "IdleTimeout::expiry@{now:?} pto={pto:?} => {t:?}",
             t = start + delay
         );
         start + delay
```

Comment:
Looking at this, perhaps we can simplify:

```rust
let t = start + delay;
qtrace!("IdleTimeout::expiry@{now:?} pto={pto:?} => {t:?}");
t
```

Or with the above factoring...

```rust
let t = self.start() + delay;
qtrace!("IdleTimeout::expiry@{now:?} pto={pto:?} => {t:?}");
t
```

Reply:
Will do.